codegens/nodejs-request/lib/request.js

```diff
@@ -24,9 +30,19 @@ function makeSnippet (request, indentString, options) {
     });
   }
   if (isFormDataFile) {
-    snippet += 'var fs = require(\'fs\');\n';
+    if (options.ES6_enabled) {
+      snippet += 'let fs = require(\'fs\');\n';
```

Comment:
`const` instead of `let`

Reply:
@shreys7 Last time, I used const so you asked me to use var/ let. That's why I changed.

Reply:
no. Those changes for if-else statements. We should use const for fs and request.